dist/dattobd.spec

```diff
@@ -219,9 +219,15 @@ Group:           System Environment/Kernel
 BuildArch:       noarch
 %endif
 
+%if 0%{?rhel} >= 6 || 0%{?fedora} >= 23 || 0%{?suse_version} >= 1315
+Requires(preun): dkms >= 2.3.0
+Requires:        dkms >= 2.3.0
+Requires(post):  dkms >= 2.3.0
```

Comment:
Lower the version to `2.3`, as otherwise it won't match:

```
$ rpmdev-vercmp 2.3 2.3.0
2.3 < 2.3.0
```

Reply:
Changed to `2.3`